app/scripts/migrations/137.test.ts

```diff
@@ -516,8 +497,194 @@ describe('migration #137', () => {
           domains: {},
         },
       });
+      const basePermission = {
+        id: '1',
+        date: 2,
+        invoker: 'test.com',
+        parentCapability: PermissionNames.eth_accounts,
+      };
       const currentScope = `eip155:${chainId}`;
 
+      it('skips eth_accounts and permittedChains permissions when they are missing an id', async () => {
```

Comment:
```suggestion
      it('skips eth_accounts and permittedChains permissions when they are missing metadata', async () => {
```

maybe? idk

Reply:
yeah... didn't want to do all the permuations of this (id, date, invoker) but sure lets just call it metadata